docker-compose-ssl.yml

```diff
@@ -0,0 +1,95 @@
+version: '3'
+services:
+  nginx-proxy:
+    image: jwilder/nginx-proxy
+    container_name: nginx-proxy
+    ports:
+      - "80:80"
```

Comment:
Do we need these mapping:
```
80:80
443:443
```

Reply:
Yes, this maps the host port to the container port.